tdrs-backend/tdpservice/parsers/validators.py

```diff
@@ -598,3 +598,120 @@ def validate_header_rpt_month_year(datafile, header, generate_error):
             field=None,
         )
     return is_valid, error
+
+class CatFourValidator:
```

Comment:
nitpick but do we have a better name for the validator class/functions than "cat 4"? maybe `CaseConsistencyValidator` or something to that effect?

Reply:
I like that much better!